test/helpers/user_helpers.rb

```diff
@@ -11,14 +11,6 @@ def mock_login(id: nil, uid: nil, role: nil)
       ApplicationController.any_instance.stubs(:current_user).returns(user)
     end
 
-    def wait_for_jQuery(secs = Capybara.default_max_wait_time)
-      ActiveSupport::Notifications.instrument 'cmrdash.performance', activity: 'Helpers::AjaxHelpers#wait_for_jQuery' do
-        Timeout.timeout(secs) do
-          loop until finished_all_jQuery_requests?
-        end
-      end
-    end
-
     def finished_all_jQuery_requests?
```

Comment:
I think I need to remove this too actually, don't think it is used anywhere anymore.